src/sentry/pipeline/__init__.py

```diff
@@ -1,5 +1,5 @@
-from .base import Pipeline
-from .provider import PipelineProvider
+from .provider import PipelineProvider  # isort:skip
```

Comment:
In `src/sentry/integrations/base.py`, we do 
```py
from sentry.pipeline import PipelineProvider
```
which goes here and imports these submodules in order. But because `src/sentry/pipeline/base.py` does 
```py
from . import PipelineProvider
```
There would be a circular dependency.